Osiris/Hacks/Visuals.cpp

```diff
@@ -406,3 +406,32 @@ void Visuals::skybox(FrameStage stage) noexcept
         memory->loadSky(sv_skyname->string);
     }
 }
+
+void Visuals::xCrosshair() noexcept
+{
+    static auto crosshairalpha = interfaces->cvar->findVar("cl_crosshairalpha");
+    crosshairalpha->setValue(255);
+
+    if (!config->visuals.xCrosshair.enabled)
+        return;
+
+    crosshairalpha->setValue(0);
+
+    if (!interfaces->engine->isInGame() || !localPlayer.get()->isAlive())
+        return;
+    
+        const auto [width, height] = interfaces->surface->getScreenSize();
+
+        const auto width_mid = width / 2;
+        const auto height_mid = height / 2;
+ 
+        if (config->visuals.xCrosshair.rainbow)
```

Comment:
so if crosshair rainbow is enabled, then set the draw color to rainbow but not draw the crosshair?

Reply:
wdym it draws